ai-worker/app/resumes/api/controller.py

```
@@ -0,0 +1,47 @@
+
+        # 3. DB 저장 (DB 커넥션 및 트랜잭션 처리)
+        # resume_repo.update_resume_data(request.id, summary, vector) # TODO: DB연결 설정 후 주석 해제
+        resume_repo.update_resume_data(request.id, summary, vector)
+
+        # 4. 결과 반환
+        return AIProcessResult(
+            id=str(request.id),
+            summary=summary,
+            vector_status="generated"
+        )
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)